code/src/WijmoProvider/Columns/AbstractProviderColumn.ts

```diff
@@ -90,6 +77,10 @@ namespace WijmoProvider.Column {
 
             const providerGrid: wijmo.grid.FlexGrid = this.grid.provider;
 
+            if (this.config.conditionalFormat) {
```

Comment:
You now have the method -> hasConditionalFormat available because it is a public getter inside the AbstractColumn

Reply:
agreed